support-frontend/assets/helpers/checkoutForm/checkoutForm.ts

```diff
@@ -62,15 +62,17 @@ export const invalidReason = (form: Record<string, any> | null): string => {
 
 		return invalidReasonString;
 	} catch (e) {
-		return e;
+		return 'unknown';
```

Comment:
I don't think we need this try/catch anymore (or maybe even this function at all?). But for now we just need to ensure it returns a string from the catch block